crnlib/crn_file_utils.cpp

```diff
@@ -216,11 +213,11 @@ bool file_utils::is_path_or_drive_separator(char c) {
 #endif
 }
 
-bool file_utils::is_drive_separator(char c) {
 #ifdef WIN32
+bool file_utils::is_drive_separator(char c) {
   return (c == ':');
 #else
-  c;
+bool file_utils::is_drive_separator(char /* c */) {

```

Comment:
Using `(void)x` seems nicer here instead of writing the function declaration twice